tests/SampleAppCRT/enc/CMakeLists.txt

```diff
@@ -5,7 +5,7 @@
 
 oeedl_file(../SampleAppCRT.edl enclave gen)
 
-add_enclave(TARGET SampleAppCRT SOURCES SampleAppCRT.cpp ${gen})
+add_enclave(TARGET SampleAppCRT UUID 31c4ecec-b94a-432a-ab76-4df401b6ee59 SOURCES SampleAppCRT.cpp ${gen})
```

Comment:
This seems like an undue burden for the test designers. Consider adding a file to the source directory called uuid.txt. The **add_enclave** cmake function can detect the existence of this file and do two things:
- Generate the appropriate C-header file (shown below).
- Name the executable accordingly.

Reply:
Implemented `OE_SET_ENCLAVE_OPTEE`. See [this commit](https://github.com/openenclave/openenclave/pull/1820/commits/4f06e4da250098e8becb794fdb52e9e43e1f34aa].